recipes/pyspark-flame/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "pyspark-flame" %}
+{% set version = "0.2.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 807099c77fbaefaeb71217995bd22d203c697c6ce32c8e75ff53e1b1b56d9322
+
+build:
+  number: 0
```

Comment:
Can this be `noarch: python` ?

xref: https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python

Reply:
Done.

Reply:
@nehaljwani I don't believe this is possible, since pyspark requires python <3.8.